custom_components/ha-glocaltokens/sensor.py

```diff
@@ -43,3 +62,92 @@ def icon(self):
     def device_class(self):
         """Return de device class of the sensor."""
         return "glocaltokens__custom_device_class"
+
+class GlocaltokensAlarmSensor(GlocaltokensEntity):
+    """Representation of a Sensor."""
+
+    def __init__(self, coordinator, entry, device):
+        """Initialize the sensor."""
+        super().__init__(coordinator, entry)
+        self._name = device[GLOCALTOKENS_DEVICE_NAME]
+        self._token = device[GLOCALTOKENS_TOKEN]
+        self._alarms = device[GLOCALTOKENS_ALARMS]
+
+        if len(self._alarms) > 0:
+            self._state = self._alarms[0]['fire_time'] / 1000
```

Comment:
Same, it would be nice to have all those key words like `fire_time` as a constant, while having this conversion ` / 1000` as a separate method. We can even add a whole separate file like `utils.py` with reusable methods that we can access across the integration

Reply:
i think that is a good idea to separate it into utils.py